cidc_api/models.py

```diff
@@ -105,11 +105,16 @@ def get_DOMAIN() -> dict:
     domain["new_users"]["resource_methods"] = ["POST"]
 
     # Restrict operations on resources that only admins should be able to access
-    for resource in ["users", "permissions", "trial_metadata"]:
+    for resource in ["users", "trial_metadata"]:
         domain[resource]["allowed_roles"] = [CIDCRole.ADMIN.value]
         domain[resource]["allowed_item_roles"] = [CIDCRole.ADMIN.value]
 
-    # Make permissions deletable by admins
+    # Restrict operations on the 'permissions' resource:
+    # * Only admins can write or update 'permissions'
+    # * All users can read permissions, but the results will be filtered by
+    #   services.permissions.update_permission_filters to only include their permissions
+    domain["permissions"]["allowed_write_roles"] = [CIDCRole.ADMIN.value]
+    domain["permissions"]["allowed_item_roles"] = [CIDCRole.ADMIN.value]
```

Comment:
I see
>* All users can read permissions, but the results will be filtered by

but then only
> domain["permissions"]["allowed_item_roles"] = [CIDCRole.ADMIN.value]

Am I missing something ?

Reply:
Eve distinguishes between resource level requests to, e.g., `/permissions`, and item-level requests to `/permissions/<item id>`. There isn't a reason to give non-admins item-level access, so that's line 117. I can update the comment, though.

Reply:
^this is why a non-admin user gets 200 from a GET to `/permissions` in test_auth.py